app/assets/stylesheets/application.css.scss

```diff
@@ -288,6 +299,11 @@ input[type="checkbox"].toggle { margin-top: 0; }
 }
 .form-group {
   margin-top: 20px;
+  margin-left: 0px !important;
+  padding-right: 10px;
+  label{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space